libbpfgo.go

```diff
@@ -327,10 +327,10 @@ func (m *Module) BPFLoadObject() error {
 
 func (m *Module) GetMap(mapName string) (*BPFMap, error) {
 	cs := C.CString(mapName)
-	bpfMap := C.bpf_object__find_map_by_name(m.obj, cs)
+	bpfMap, errno := C.bpf_object__find_map_by_name(m.obj, cs)
```

Comment:
This API function is already returning just NULL on error with error code in errno.

Reply:
Cool, I came exactly because of that. Was going to ask why those 2 calls were different than all others, having errno being set directly in cgo's call. Thanks.